x-pack/plugins/security_solution/public/common/components/events_viewer/index.tsx

```diff
@@ -221,7 +222,7 @@ const StatefulEventsViewerComponent: React.FC<Props> = ({
       <DetailsPanel
         alertConsumers={alertConsumers}
         browserFields={browserFields}
-        entityType={EntityType.ALERTS}
```

Comment:
Detail panel should use whatever entityType is passed in for hosts vs alerts page